webapp/src/Service/ImportExportService.php

```diff
@@ -577,25 +653,93 @@ protected function importGroupsTsv(array $content, string &$message = null): int
             } else {
                 $action = EventLogService::ACTION_UPDATE;
             }
-            $teamCategory->setName($groupItem['name']);
+            $teamCategory
+                ->setName($groupItem['name'])
+                ->setVisible($groupItem['visible'] ?? true)
+                ->setSortorder($groupItem['sortorder'] ?? 0);
             $this->em->flush();
             if ($contest = $this->dj->getCurrentContest()) {
-                $this->eventLogService->log('team_category', $teamCategory->getCategoryid(), $action,
-                                            $contest->getCid());
+                $this->eventLogService->log('team_category',
+                    $teamCategory->getCategoryid(), $action,
+                    $contest->getCid());
```

Comment:
Nitpick, but *if* you're reindenting, then I think *all* arguments should go onto indented lines and the final `);` on a separate line (following PSR1, although I really don't like putting each single argument on a line of its own).